tests/integration/kubernetes/k8s-attach-handlers.bats

```diff
@@ -15,28 +15,39 @@ setup() {
 	pod_name="handlers"
 
 	get_pod_config_dir
-}
+	yaml_file="${pod_config_dir}/test-lifecycle-events.yaml"
 
-@test "Running with postStart and preStop handlers" {
 	# Create yaml
 	sed -e "s/\${nginx_version}/${nginx_image}/" \
-		"${pod_config_dir}/lifecycle-events.yaml" > "${pod_config_dir}/test-lifecycle-events.yaml"
+		"${pod_config_dir}/lifecycle-events.yaml" > "${yaml_file}"
+
+	# Add policy to yaml
```

Comment:
This block could be in a `setup_file()` which is executed once. Unless you really need the policy files re-generate for every and each test case.

Another question is: are you planning to copy this boilerpate over all the k8s tests as you go enabling them? or maybe run 1x on https://github.com/kata-containers/kata-containers/blob/main/tests/integration/kubernetes/setup.sh ?

Reply:
The steps to generate Policy are specific to each test case, so my plan is to add similar steps to every relevant bats file. 

Examples why the steps are specific to each test case:
- Some of the tests use an already existing YAML file vs. generating one with sed
- Some of the tests want to "kubectl exec" one or more commands, others don't exec anything
- Some use config maps, while others don't 
- etc.

So, I don't think the centralized setup.sh approach works for these tests.